spec/components/chat_entry_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+describe "Chat entry", type: :view do
+  def component_name
+    "chat_entry"
+  end
+
+  it "renders the chat entry component" do
+    render_component({
+      href: "/get-started",
+      heading_text: "Try GOV.UK Chat",
+      description_text: "Find answers to your business questions",
+    })
+
+    assert_select ".gem-c-chat-entry"
+    assert_select "h2.gem-c-chat-entry__heading [href]", href: "/get-started", text: "Try GOV.UK Chat"
+    assert_select ".gem-c-chat-entry__description .govuk-body", text: "Find answers to your business questions"
+  end
+
+  it "renders the chat entry component with a different heading level" do
+    render_component({
+      heading_level: 3,
+      href: "/get-started",
+      heading_text: "Try GOV.UK Chat",
+      description_text: "Find answers to your business questions",
+    })
+
+    assert_select "h3.gem-c-chat-entry__heading [href]", href: "/get-started", text: "Try GOV.UK Chat"
+  end
+
+  it "renders the chat entry component with a border top" do
+    render_component({
+      href: "/get-started",
+      heading_text: "Try GOV.UK Chat",
+      description_text: "Find answers to your business questions",
+      border_top: true,
+    })
+
+    assert_select ".gem-c-chat-entry.gem-c-chat-entry--border-top"
+  end
```

Comment:
Quick question - do we need a companion test for this to check the component renders with a bottom border when passed `border_bottom: true`?

Reply:
Yep, I think that's a good idea. It's now been added.

Reply:
Awesome, thanks Jon. Just one more quick thing (sorry I just noticed this) - do we need to add a corresponding section in the component guide doc for the `border_bottom` option?

Reply:
Yep sounds good to me. I've added a separate test now.